replica/database.cc

```diff
@@ -2424,7 +2424,10 @@ future<> database::flush(const sstring& ksname, const sstring& cfname) {
 
 future<> database::flush_table_on_all_shards(sharded<database>& sharded_db, table_id id) {
     return sharded_db.invoke_on_all([id] (replica::database& db) {
-        return db.find_column_family(id).flush();
+        if (db.column_family_exists(id)) {
```

Comment:
But this method is called with table_id at hand, which means that it was found by the caller which, in turn, means that the table in question does exist. How come it's not any longer find-able here?

Reply:
The callers do not necessarily find the id right before calling `flush_table_on_all_shards` and so it might be dropped in between